projects/dendrites/gaussian_classification/run_dendritic_network.py

```diff
@@ -320,25 +326,30 @@ def evaluate_model(exp):
 
     # --------------------------- CONTINUAL LEARNING PHASE -------------------------- #
 
+    # Evaluation can be slow. Only run evaluation when training task_id is in this list
+    eval_tasks = [num_tasks - 2, num_tasks - 1]
     for task_id in range(num_tasks):
 
         # Train model on current task
+        t1 = time.time()
         exp.train_loader.sampler.set_active_tasks(task_id)
         for _ in range(num_epochs):
             train_model(exp)
-
-        print(f"=== AFTER TASK {task_id} ===")
-        print("")
+        t2 = time.time()
 
         # Evaluate model accuracy on each task separately
-        for eval_task_id in range(task_id + 1):
+        if task_id in eval_tasks:
+            for eval_task_id in range(task_id + 1):
+
+                exp.val_loader.sampler.set_active_tasks(eval_task_id)
+                acc_task = evaluate_model(exp)
+                if isinstance(acc_task, tuple):
+                    acc_task = acc_task[0]
 
-            exp.val_loader.sampler.set_active_tasks(eval_task_id)
-            acc_task = evaluate_model(exp)
-            if isinstance(acc_task, tuple):
-                acc_task = acc_task[0]
+                print(f"task {eval_task_id} accuracy: {acc_task}")
 
-            print(f"task {eval_task_id} accuracy: {acc_task}")
+        t3 = time.time()
+        print(f"=== AFTER TASK {task_id} === TRAIN TIME: {t2-t1}, EVAL TIME: {t3-t2}")
```

Comment:
This print statement (i.e. 'AFTER TASK ...') was originally before the evaluation loop so that it's clear all the printed accuracies are computed after task X, but now nothing is printed before the evaluation loop. Can you move the print statement back there?

Reply:
OK, I split it into two so that we can also show the evaluation time